metagraph/src/cli/align.cpp

```diff
@@ -84,7 +83,7 @@ template std::unique_ptr<IDBGAligner> build_aligner<DeBruijnGraph>(const DeBruij
 
 void map_sequences_in_file(const std::string &file,
                            const DeBruijnGraph &graph,
-                           const DBGSuccinct *dbg,
+                           const DBGSuccinct &dbg,
```

Comment:
why do we pass two graphs?

Reply:
I don't remember why we set it up this way, it's essentially just a dynamic cast of the first graph.

Reply:
I don't see why we need to pass both

Reply:
We actually don't, so I just got rid of it and added another dynamic cast into this function.